docker-compose.yml

```diff
@@ -22,10 +22,10 @@ services:
     environment:
        XDEBUG_CONFIG: "remote_host=172.17.0.1 remote_enable=1" 
        DB_DRIVER_DEVELOPMENT: "mysql"
-       DB_HOST_DEVELOPMENT: "db"
-       DB_DATABASE_DEVELOPMENT: "dashboard"
-       DB_USERNAME_DEVELOPMENT: "root"
-       DB_PASSWORD_DEVELOPMENT: "root"
+       DB_HOST_DEVELOPMENT: "72.29.77.223"
+       DB_DATABASE_DEVELOPMENT: "givercardwebcom_voip"
+       DB_USERNAME_DEVELOPMENT: "givercardwebcom_voip"
+       DB_PASSWORD_DEVELOPMENT: "@VoxlinkDP"
```

Comment:
remove username and password and put in .env file

Reply:
created